docs/concepts/pdp/configuration.mdx

```diff
@@ -39,15 +39,29 @@ The URL of the Permit.io control plane.
 
 Default: `None`
 
-PDP debug mode configuration.
-There available configurations are:
-- `PDP_DEBUG=None` - The default behavior, will use the configuration set by Permit.io Cloud.
+Enables debug mode, which injects debug attributes to OPA policy queries.
+The available configurations are:
+- `PDP_DEBUG=None` - The default behavior - will use the configuration set by Permit.io Cloud.
 - `PDP_DEBUG=True` - Will enable Debug Mode for this specific PDP.
 - `PDP_DEBUG=False` - Will disable Debug Mode for this specific PDP.
 
 For more detailed configuration about the PDP Debug Mode,
 please check the full documentation [here](/how-to/use-audit-logs/debug-mode/).
 
+#### PDP_PORT
+
+Default: `7766`
+
+The port the PDP server will listen on.
+
+#### PDP_USE_NEW_AUTHORIZED_USERS
```

Comment:
this should be EAP or deprecated in some way because we will change this to be fully managed by the control plane one the policy generation process new design is implemented

Reply:
I've wrote there that this feature is controlled by the control plane - like we do in other configs like OPAL_SERVER_URL